app/helpers/turbo/frames_helper.rb

```diff
@@ -42,6 +42,6 @@ def turbo_frame_tag(*ids, src: nil, target: nil, **attributes, &block)
     id = ids.first.respond_to?(:to_key) ? ActionView::RecordIdentifier.dom_id(*ids) : ids.join('_')
     src = url_for(src) if src.present?
 
-    tag.turbo_frame(**attributes.merge(id: id, src: src, target: target).compact, &block)
+    content_tag(:turbo_frame, nil, **attributes.merge(id: id, src: src, target: target).compact, &block)
```

Comment:
According to the [content_tag Rails API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag), the use of `content_tag` is considered legacy syntax.